### ISSUE TITLE
PCA analysis Show side By Side plots

### ISSUE THREAD

**Issue by ethender** [ethender/MastersProject#3]
Training set and Test set should be same row and columns. Try to accomplish using subplots
